core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/script/dto/version/ScriptVersionDto.java

```diff
@@ -15,4 +16,19 @@
     private long number;
     private String description;
 
+    public long getNumber() {
+        return number;
+    }
+
+    public void setNumber(long number) {
+        this.number = number;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
```

Comment:
methods should not be added. The lombol @Data annotation creates these methods behind the scene